apps/vx-scan/frontend/src/app_root.tsx

```diff
@@ -283,7 +283,7 @@ export function AppRoot({
   const unconfigureServer = useCallback(
     async (options: { ignoreBackupRequirement?: boolean } = {}) => {
       try {
-        await config.setElection(undefined, options);
+        await apiClient.unconfigureElection(options);
```

Comment:
Nice, I like that we're separating setting and removal instead of overloading setting to handle both cases